virttest/qemu_storage.py

```diff
@@ -348,7 +349,7 @@ def snapshot_del(self, blkdebug_cfg=""):
         else:
             cmd += " %s" % self.image_filename
 
-        process.system_output(cmd)
+        return process.run(cmd, ignore_status=True)
 
```

Comment:
@PandaWei Thanks for reminding. update the patch to reduce the impact for positive test.